scripts/apps/search/directives/ItemList.js

```diff
@@ -718,11 +718,11 @@ export function ItemList(
                                 ));
 
                     var elem = React.createElement('div',
-                        {
-                            className: 'dropdown__menu more-activity-menu',
-                            onMouseOver: this.toggle,
-                            onMouseLeave: this.close
-                        },
+                                                   {
```

Comment:
Hi Gabriel, I got this error with lint. Hence added the indentations as lint expected.
![ss](https://cloud.githubusercontent.com/assets/13354440/21119862/990db546-c118-11e6-9100-f0ea61c1a6e4.jpg)